doc/script_commands.txt

```diff
@@ -3304,29 +3304,29 @@ to by its map name. Castle information is stored in `guild_castle` SQL table.
 
 Types of data correspond to `guild_castle` table columns:
 
- 1 - `guild_id`   - Guild ID.
- 2 - `economy`    - Castle Economy score.
- 3 - `defense`    - Castle Defense score.
- 4 - `triggerE`   - Number of times the economy was invested in today.
- 5 - `triggerD`   - Number of times the defense was invested in today.
- 6 - `nextTime`   - unused
- 7 - `payTime`    - unused
- 8 - `createTime` - unused
- 9 - `visibleC`   - Is 1 if a Kafra was hired for this castle, 0 otherwise.
-10 - `visibleG0`  - Is 1 if the 1st guardian is present (Soldier Guardian)
-11 - `visibleG1`  - Is 1 if the 2nd guardian is present (Soldier Guardian)
-12 - `visibleG2`  - Is 1 if the 3rd guardian is present (Soldier Guardian)
-13 - `visibleG3`  - Is 1 if the 4th guardian is present (Archer Guardian)
-14 - `visibleG4`  - Is 1 if the 5th guardian is present (Archer Guardian)
-15 - `visibleG5`  - Is 1 if the 6th guardian is present (Knight Guardian)
-16 - `visibleG6`  - Is 1 if the 7th guardian is present (Knight Guardian)
-17 - `visibleG7`  - Is 1 if the 8th guardian is present (Knight Guardian)
+CD_GUILD_ID    - Guild ID.
+CD_ECONOMY     - Castle Economy score.
+CD_DEFENSE     - Castle Defense score.
+CD_TRIGGER_E   - Number of times the economy was invested in today.
+CD_TRIGGER_D   - Number of times the defense was invested in today.
+CD_NEXT_TIME   - unused
+CD_PAY_TIME    - unused
+CD_CREATE_TIME - unused
+CD_VISIBLE_C   - Is 1 if a Kafra was hired for this castle, 0 otherwise.
+CD_VISIBLE_G0  - Is 1 if the 1st guardian is present (Soldier Guardian)
```

Comment:
We could go ahead with more explicit constant name for readability. I don't think it will be problematic if it doesn't match the name from castle sql
Proposal:
```
CD_CURRENT_ECONOMY
CD_CURRENT_DEFENSE
CD_VISIBLE_KAFRA
CD_VISIBLE_GUARDIAN_0
```

Reply:
any better idea for guardian so it can allow for guardianN?
I planned to change the castle table actually, to get rid the table fields for guardian.